docs/_documentation/presenters/mac-view-presenter.md

```diff
@@ -59,6 +59,19 @@ The presentation can be customized through this properties:
 - TabTitle: Defines the title of the tab that will be displayed in the segmented control for tabs.
 - WindowIdentifier: identifier for the window where the view should be displayed.
 
+### MvxStoryboardWindowPresentationAttribute
```

Comment:
I think this documentation needs to be merged into the old attribute now we will keep that.

Reply:
ah yes good catch

Reply:
i changed the docs